framer.go

```diff
@@ -16,11 +16,20 @@ const (
 	maxControlFrames = 16 << 10
 )
```

Comment:
Can we remove this and `framer.queuedTooManyControlFrames`?

Reply:
Not quite yet. There are still a few other frame types that use the control frame queue: 
* connection ID management: NEW_CONNECTION_ID, RETIRE_CONNECTION_ID
* stream limit: MAX_STREAMS, STREAMS_BLOCKED
* connection-level flow control: MAX_DATA and the corresponding BLOCKED frame
* some frames that are sent very rarely: NEW_TOKEN, HANDSHAKE_DONE

The plan is to make sending of these frames more explicit. This seems more tractable now that the streams don't use the frame queue anymore.